src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs

```diff
@@ -201,6 +204,58 @@ public void SelectCollectionAndClose(string path)
 			}
 		}
 
+		/// <summary>
+		/// Path to the basic user configuration information file for Dropbox.  It may or may not exist.
+		/// We make it a static variable because we only want to work at getting the Dropbox folder once
+		/// and this make a suitable flag for doing that.
+		/// </summary>
+		private static string _dropboxInfoFile;
+		/// <summary>
+		/// Path to the user's Dropbox folder.  It is static because we only want to look it up once.
+		/// </summary>
+		private static string _dropboxFolder;
+		/// <summary>
+		/// This method checks 'path' for being in the Dropbox folder.  If so, it displays a warning
+		/// message and returns true.
+		/// </summary>
+		/// <returns>true if 'path' is in the Dropbox folder</returns>
+		internal static bool CheckForBeingInDropboxFolder(string path)
+		{
+			if (_dropboxInfoFile == null)
+			{
+				var baseFolder = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+				if (Palaso.PlatformUtilities.Platform.IsWindows)
+					_dropboxInfoFile = Path.Combine(baseFolder, "Dropbox\\info.json");
+				else
+					_dropboxInfoFile = Path.Combine(Path.GetDirectoryName(baseFolder), ".dropbox/info.json");
+				if (!File.Exists(_dropboxInfoFile))
+					return false;	// User must not have Dropbox installed
+				var info = File.ReadAllText(_dropboxInfoFile);
+				var matches = Regex.Matches(info, @"{""path"": ""([^""]+)"",");
+				if (matches.Count > 0)
+				{
+					_dropboxFolder = matches[0].Groups[1].Value;
```

Comment:
Steve, it looks like this will only look at the first entry.  Here's my info.json, which has two entries:
{"personal": {"path": "X:\\Dropbox (Personal)", "host": 1630633661}, "business": {"path": "X:\\SILLEAD Dropbox\\Dropbox (SIL LEAD)", "host": 1479395800}}